### DISCUSSION TITLE
Is there a way to change the settings of Widget Style Qt Style "Fusion"?

### DISCUSSION THREAD

**Discussion by ralf3u** [lxqt/lxqt#2639] [Q&A] [answered]
I would like to have the focused tabs in PCManFM-Qt with a underline with Qt Style `Fusion`. It should be possible, because in Lubuntu 24.04 with the Qt Style `Cleanlooks` all not-focused tabs are underlined.

Is there somewhere the possibility to change Widget Style the Qt Style "Fusion"? Or rather: Where is `Fusion` located, so maybe I can change CSS there?

Thank you in advance for an answer.

**Comment by tsujan** [comment, marked as answer]
The answer is "no". Fusion doesn't have any settings. You can only change its colors in LXQt Appearance Configuration.

> Where is Fusion located, so maybe I can change CSS there?

Fusion is a widget style plugin; it doesn't have any CSS.

**Comment by ralf3u** [comment]
Thank you for your help.
